cmd/xl-v1-utils.go

```diff
@@ -42,14 +41,20 @@ func reduceErrs(errs []error, ignoredErrs []error) error {
 		errorCounts[err]++
 	}
 	max := 0
-	var errMax error
 	for err, count := range errorCounts {
 		if max < count {
 			max = count
-			errMax = err
+			maxErr = err
+		}
+	}
+	if maxErr != nil {
+		if max >= quorum {
+			maxErr = traceError(maxErr, errs...)
+		} else {
+			maxErr = traceError(errXLReadQuorum, errs...)
```

Comment:
It could be read or write quorum depending on the operation. It can't be errXLReadQuorum unconditionally.